fnet/data/multichtiffdataset.py

```diff
@@ -9,35 +9,40 @@
 class MultiChTiffDataset(FnetDataset):
     """Dataset for multi-channel tiff files.
 
-    Currently assumes that images are loaded in ZCXY format
+    Currently assumes that images are loaded in STCZYX format
 
     """
 
-    def __init__(self, dataframe: pd.DataFrame = None, path_csv: str = None,
-                 transform_signal=None,
-                 transform_target=None):
+    def __init__(
+        self,
+        dataframe: pd.DataFrame = None,
+        path_csv: str = None,
+        transform_signal=None,
+        transform_target=None,
+    ):
 
         super().__init__(dataframe, path_csv, transform_signal, transform_target)
 
-        self.df['channel_signal'] = [int(ch)
-                                     for ch in self.df['channel_signal']]
-        self.df['channel_target'] = [int(ch)
-                                     for ch in self.df['channel_target']]
+        self.df["channel_signal"] = [int(ch) for ch in self.df["channel_signal"]]
+        self.df["channel_target"] = [int(ch) for ch in self.df["channel_target"]]
 
-        assert all(i in self.df.columns for i in [
-                   'path_tiff', 'channel_signal', 'channel_target'])
+        assert all(
+            i in self.df.columns
+            for i in ["path_tiff", "channel_signal", "channel_target"]
+        )
 
     def __getitem__(self, index):
         element = self.df.iloc[index, :]
-        has_target = not np.isnan(element['channel_target'])
+        has_target = not np.isnan(element["channel_target"])
 
-        im_tmp = tifffile.imread(element['path_tiff'])
+        # aicsimageio.imread loads as STCZYX, so we load only CZYX
+        im_tmp = imread(element["path_tiff"])[0, 0]
```

Comment:
I would actually change this to:
```python
with AICSImage(element["path_tiff"]) as img_read:
    im_tmp = img.get_image_data(S=0, T=0)
```

This returns the same `np.ndarray` as expected but doesn't rely on a specific dim order.

Reply:
didn't work